chirpr.py

```diff
@@ -119,9 +128,37 @@ def chirps():
     return render_template('admin/chirps.html', chirps=chirp_list)
     
     
+@app.route('/chirps/<handle>')
+def chirps_hn(handle):
+    uid = db_access.get_id(handle)
+    return redirect('%s?filter=%s'%(url_for('chirp'), handle))
+
+    
+@app.route('/chirp', methods=['GET', 'POST'])
+def chirp():
+    if 'user' in session:
+        user = session['user']
+        MD = False
+        if request.method == 'POST':
+            content = request.form["content"]
+            if 'MD' in request.form:
+                if request.form['MD'] == 'on':
+                    content = markdown2.markdown(content)
```

Comment:
You can use

   md =  request.form.get('MD', None)

and `md` will get the falue of `MD` or `None` if it is not in the form. This method prevents the need for nested ifs.